smoke_tests/src/test/groovy/uk/gov/dhsc/htbhf/smoke/ServiceIsRunningSpec.groovy

```diff
@@ -0,0 +1,30 @@
+package uk.gov.dhsc.htbhf.smoke
+
+import groovyx.net.http.RESTClient
+import spock.lang.Shared
+import spock.lang.Specification
+
+class ServiceIsRunningSpec extends Specification {
```

Comment:
Could we change this to be junit5 instead of groovy? You can copy the code from smart-stub. I think it'll be nice not to use another language (especially groovy).